Content.Client/Administration/UI/Logs/AdminLogsEui.cs

```diff
@@ -100,36 +104,41 @@ private async void ExportLogs()
 
         try
         {
-            await using var writer = new StreamWriter(file.Value.fileStream);
+            // Buffer is set to 4KB for performance reasons. As the average export of 1000 logs is ~200KB
+            await using var writer = new StreamWriter(file.Value.fileStream, bufferSize: 4096);
+            await writer.WriteLineAsync(CsvHeader);
             foreach (var child in LogsControl.LogsContainer.Children)
             {
                 if (child is not AdminLogLabel logLabel || !child.Visible)
                     continue;
 
                 var log = logLabel.Log;
 
+                // Date
                 // I swear to god if someone adds ,s or "s to the other fields...
                 await writer.WriteAsync(log.Date.ToString("s", System.Globalization.CultureInfo.InvariantCulture));
-                await writer.WriteAsync(',');
+                await writer.WriteAsync(CsvSeparator);
+                // ID
                 await writer.WriteAsync(log.Id.ToString());
-                await writer.WriteAsync(',');
-                await writer.WriteAsync(log.Impact.ToString());
-                await writer.WriteAsync(',');
-                // Message
-                await writer.WriteAsync('"');
-                await writer.WriteAsync(log.Message.Replace("\"", "\"\""));
-                await writer.WriteAsync('"');
-                // End of message
-                await writer.WriteAsync(',');
-
+                await writer.WriteAsync(CsvSeparator);
+                // PlayerID
                 var players = log.Players;
                 for (var i = 0; i < players.Length; i++)
                 {
                     await writer.WriteAsync(players[i] + (i == players.Length - 1 ? "" : " "));
```

Comment:
```suggestion
                    await writer.WriteAsync((i == 0 ? "" : " ") + players[i]);
```

I believe this is slightly more efficient because it's checking against a constant instead.
Will it ever matter? Who knows.
Will C# even optimise it enough for it to matter? Who knows.